tox.ini

```diff
@@ -1,43 +1,28 @@
 [tox]
 distribute = False
 envlist =
-    py{27,34,35}-django{18,110}
-    py{27,34,35,36}-django111
-    py{34,35,36,37}-django{20}
-    py{35,36,37}-django{21,22}
+    py{35,36,37}-django{22}
+    py{36,37,38}-django{30}
     docs
 skip_missing_interpreters = True
 
 [travis]
 python =
-    2.7: py27
-    3.4: py34
     3.5: py35
     3.6: py36, docs
     3.7: py37
+    3.8: py38
 
 [testenv]
 setenv = DJANGO_SETTINGS_MODULE=easy_thumbnails.tests.settings
 usedevelop = True
 deps =
-    django18: Django>=1.8,<1.9
-    django19: Django>=1.9,<1.10
-    django110: Django>=1.10,<1.11
-    django111: Django>=1.11,<2.0
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
-    django22: Django>=2.2a1,<3.0
+    django22: Django>=2.2,<3.0
+    django30: Django>=3.0,<3.1
     testfixtures
```

Comment:
How about append tests for actual django version like this
`djangomaster: https://github.com/django/django/archive/master.tar.gz
`
This will allow to monitor the current status with the current Django, which has not yet been released.
For djangomaster we can set allow_failures in travis

Reply:
Seems out of scope for this PR, maybe you can do a follow up PR with this suggestion after this is merged?

Reply:
NP. Just a suggestion